lib/proxy/plugin_initializer.rb

```diff
@@ -254,7 +265,7 @@ def load_settings(main_plugin_settings = {})
 
     plugin.settings = ::Proxy::Settings::Plugin.new({}, settings)
 
-    log_used_settings(settings)
+    send(how_to_log, settings)
```

Comment:
Wouldn't `yield if block_given` and mapping the log statements via `block` be better here?

Reply:
Replaced with a block.